R/gcalendr_auth.R

```diff
@@ -72,7 +69,7 @@ calendar_auth <- function(
 {
   cred <- gargle::token_fetch(
     scopes = scopes,
-    app = calendar_oauth_app(),
+    app = calendar_oauth_app() %||% calendar_app(),
```

Comment:
I think this is a better design, in terms of providing a wee bit more obfuscation of your app. We take even stronger measures in other packages.